Shared/Prefs.swift

```diff
@@ -49,6 +49,13 @@ public struct PrefsKeys {
         public static let v1MigrationCheck = "v1MigrationCheckUserPrefsKey"
     }
 
+    public struct Notifications {
+        public static let AllowAllNotifications = "AllowAllNotificationsUserPrefsKey"
```

Comment:
`AllowAllNotifications` and `TabsNotifications` can deleted as they don't exist any longer. `SyncSignInNotifications` should be renamed to `SyncNotifications`.